examples/ecr.tf

```diff
@@ -8,6 +8,8 @@ module "example_team_ecr_credentials" {
   source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.0"
   repo_name = "example-module"
   team_name = "example-team"
+  # By default scan_on_push is set to true. To disable set it to false as below:
```

Comment:
It would be good to explain what "scan_on_push" does, maybe with a link to the relevant AWS documentation page.

Reply:
done.